Examples/trame/app.py

```diff
@@ -0,0 +1,305 @@
+
+    cleaver_mesher.RemoveAllInputs()
+    renderer.RemoveActor(actor)
+
+    for fileName in state.input_file_names:
+        cleaver_mesher.AddInputData(0, input_images[fileName])
+    
+    print(state.rate_of_change)
+
+    cleaver_mesher.SetSamplingRate(state.sampling_rate)
+    cleaver_mesher.SetFeatureScaling(state.feature_scaling)
+    cleaver_mesher.SetRateOfChange(state.rate_of_change)
+    cleaver_mesher.Update()
+
+    ugrid = cleaver_mesher.GetOutput()
+    mapper.SetScalarRange(ugrid.GetCellData().GetScalars().GetRange())
+
+    renderer.AddActor(actor)
+    renderer.ResetCamera()
+
+    state.cleaver_running = False
+
+    html_view.update()
+
+
+@state.change("input_file_names")
+def update_input_available(input_file_names, **kwargs):
+    state.input_available = len(input_file_names) > 0
+    print(f"update_input_available: {state.input_available}")
+
+
+@state.change("sampling_rate")
+def update_sampling_rate(sampling_rate, **kwargs):
+    print(f"sampling_rate: {sampling_rate}")
+
+
+def reset_sampling_rate():
+    state.sampling_rate = DEFAULT_SAMPLING_RATE
+
+
+@state.change("feature_scaling")
+def update_feature_scaling(feature_scaling, **kwargs):
+    print(f"feature_scaling: {feature_scaling}")
+
+
+def reset_feature_scaling():
+    state.feature_scaling = DEFAULT_FEATURE_SCALING
+
+
+@state.change("rate_of_change")
+def update_rate_of_change(rate_of_change, **kwargs):
+    print(f"rate_of_change: {rate_of_change}")
+
+
+def reset_rate_of_change():
+    state.rate_of_change = DEFAULT_RATE_OF_CHANGE
+
+
+# -----------------------------------------------------------------------------
+# GUI Cards
+# -----------------------------------------------------------------------------
+
+compact_style = {
+    "hide_details": True,
+    "dense": True,
+}
+
+
+def ui_card(title, ui_name):
+    card_style = {}
+    title_style = {
+        "classes": "grey lighten-1 py-1 grey--text text--darken-3",
+        "style": "user-select: none; cursor: pointer",
+        **compact_style,
+    }
+    content_style = {"classes": "py-2"}
+    with vuetify.VCard(**card_style):
+        vuetify.VCardTitle(title, **title_style)
+        content = vuetify.VCardText(**content_style)
+
+    return content
+
+def input_card():
+    with ui_card("Input", "input"):
+        vuetify.VFileInput(
+            multiple=True,
+            show_size=True,
+            small_chips=True,
+            truncate_length=25,
+            v_model=("files", None),
+            dense=True,
+            hide_details=True,
+            style="max-width: 300px;",
+            accept=".nrrd,.vti",
+            __properties=["accept"],
+        )
+    
+    # TODO Add a component allowing to re-order "state.input_file_names"
+    # Possible approaches:
+    # - v-simple-table + SortableJS
+    #   See https://codepen.io/mykysyk/pen/qBdBRMB
+    # - v-chip + draggable
+    #   See https://github.com/vuetifyjs/vuetify/issues/11614 and https://jsfiddle.net/cjrqzkf0/
```

Comment:
@jourdain I would like the ability to re-order the input. Is there a component that would allow me to do so ? 

Something like this where I could drag and drop to re-order:

![image](https://user-images.githubusercontent.com/219043/163417534-dc582cf7-cc8b-4965-b593-92ab140b7294.png)

These are two approaches:

- v-simple-table + SortableJS. See https://codepen.io/mykysyk/pen/qBdBRMB
- v-chip + draggable. See https://github.com/vuetifyjs/vuetify/issues/11614 and https://jsfiddle.net/cjrqzkf0/

Do you have any recommendation ? Should I directly use functionality from pywebvue to include sortable.js ?

Reply:
No specific recommendation and in your code base you can use `enable_module` to serve some JS files and enable them in vue.js. Let me know if you need help in that JS integration.

Reply:
> you can use enable_module to serve some JS files and enable them in vue.js.

Do you happen to have an example handy ?

Reply:
https://github.com/Kitware/paraview-visualizer/blob/master/pv_visualizer/html/__init__.py